src/RNGalleryList.ts

```diff
@@ -9,7 +9,7 @@ import {DatePickerExamplePage} from './examples/DatePickerExamplePage';
 import {TimePickerExamplePage} from './examples/TimePickerExamplePage';
 //import {SketchExamplePage} from './examples/SketchExamplePage';
 import {SliderExamplePage} from './examples/SliderExamplePage';
-import {PermissionsExamplePage} from './examples/PermissionsExamplePage';
+//import {PermissionsExamplePage} from './examples/PermissionsExamplePage';
```

Comment:
For these disabled pages can you include a comment with the tracking issue number?
e.g.
```
// Disabled per issue #127
//import {PermissionsExamplePage} from './examples/PermissionsExamplePage';
```